src/config/dynamic-theme-fixes.config

```diff
@@ -216,6 +216,19 @@ div.description > span {
 
 ================================
 
+500px.com
+
+CSS
+.button.new_fav .icon .shape {
+    fill: transparent !important;
+    stroke: rgb(232, 230, 227);
+}
+.button.only_icon.hearted .icon .shape, .button.only_icon.liked .icon .shape {
```

Comment:
I'm not able to find these elements, I assume they are only available for people that are signed in?

Reply:
Yes, you can add photos in fav list only if signed in. For test try to add `hearted` class in a like button:

```html
<div class="like-button">
  <div>
    <a class="button new_fav only_icon hearted">
      <span class="value"></span>
      <svg class="icon" version="1.1" viewBox="-6.9 -13.1 40 40" x="0px" y="0px">
        <path class="shape" d="" fill="#FFFFFF" style="--darkreader-inline-fill: #e8e6e3;" data-darkreader-inline-fill=""></path>
      </svg>
    </a>
  </div>
</div>
```

Reply:
In that case..